agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -36,6 +36,46 @@
 router = APIRouter()
 
 
+@router.get(
+    "/by_resource/",
+    response_model=Any,
```

Comment:
Can you replace `Any` with `List[str]` instead? Since we know what the api router response would be.

Reply:
I initially did that, but for some reason, the code was not compiling. Do you know why this could be happening?

Reply:
Must be from this line of code:

```python
return list(map(lambda x: x.id, evaluations))
```

Reply:
Hmmm, I've replaced `Any` with `List[ObjectId]` and it worked